storage_image_import/tests/models/fake_image_relation.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2021 Camptocamp (http://www.camptocamp.com).
+# @author Iván Todorovich <ivan.todorovich@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class FakeImageRelation(models.Model):
```

Comment:
see how I used odoo-test-helper https://github.com/OCA/storage/pull/94 (to be fwd ported)